dotcom-rendering/src/palette.ts

```diff
@@ -91,6 +92,19 @@ const tableOfContentsDark = (): string => sourcePalette.neutral[86];
 const tableOfContentsBorderLight = (): string => sourcePalette.neutral[86];
 const tableOfContentsBorderDark = (): string => sourcePalette.neutral[20];
 
+const clickToViewBackgroundLight = (): string => sourcePalette.neutral[97];
+const clickToViewBackgroundDark = (): string => sourcePalette.neutral[20];
+const clickToViewBorderLight = (): string => sourcePalette.neutral[97];
+const clickToViewBorderDark = (): string => sourcePalette.neutral[46];
+const clickToViewButtonLight = (): string => sourcePalette.brand[400];
+const clickToViewButtonDark = (): string => sourcePalette.neutral[97];
+const clickToViewButtonTextLight = (): string =>
+	buttonThemeDefault.button.textPrimary;
```

Comment:
I think this is a fine pattern, but it will become wasteful once we start introducing format boundaries with #9286, as none of these values change with format.

A simple improvement may be to split into palette method signatures:
- a **static** palette, which is global and true for any and all pages
- a **format** palette, which varies in response to a given format
- a **card** palette, which varies in response to a given format and container
- a **…** palette, which varies in response to …